apps/mobile/src/components/Mint/MintCampaign/MintCampaignPreTransaction.tsx

```diff
@@ -0,0 +1,174 @@
+import { Suspense, useCallback, useEffect, useMemo, useState } from 'react';
+import { Image, View } from 'react-native';
+import { graphql, useLazyLoadQuery } from 'react-relay';
+import { contexts } from 'shared/analytics/constants';
+import { useHighlightClaimMint } from 'src/hooks/useHighlightClaimMint';
+import usePersistedState from 'src/hooks/usePersistedState';
+
+import { Button } from '~/components/Button';
+import { BaseM, BaseS, TitleLItalic, TitleS } from '~/components/Text';
+import { MintCampaignPreTransactionQuery } from '~/generated/MintCampaignPreTransactionQuery.graphql';
+
+import { MCHX_CLAIM_CODE_KEY } from './MintCampaignBottomSheet';
+
+export const MCHX_MINT_CAMPAIGN_END_DATE = '2024-05-05T10:00:00-04:00';
+
+const MCHX_COLLECTION_ID = '65ec1cefb5813a0adf4dff4e';
```

Comment:
Change this to `660d4342c6bc04d5dc5598e7`

Reply:
done

Reply:
nvm you added! lol thx